src/backend/planner/hash_join_plan.h

```diff
@@ -72,6 +72,13 @@ class HashJoinPlan : public AbstractJoinPlan {
     return std::unique_ptr<AbstractPlan>(new_plan);
   }
 
+  // Every class should implement SerializeTo method before using it.
+  // The implementation in seq_scan_plan can be referenced
+  bool SerializeTo(SerializeOutput &output) const {
+	  PL_ASSERT(&output != nullptr);
+	  throw SerializationException("This class should implement SerializeTo method");
+  }
+
```

Comment:
1. Fix tabs.
2. What is the point of the assert? The parameter is a reference, meaning you assume is it not null.